### PR TITLE
Redirection extension

### PR DESCRIPTION
**Pull request by ryan-gang**

### CODE REVIEW

**Comment by ryan-gang** [comment]
@andy1li can you go through the stage descriptions once ? See if everything makes sense ?

File: course-definition.yml [hunk 1]
```diff
@@ -751,3 +758,166 @@ stages:
...
+
+  - slug: "jv1"
+    primary_extension_slug: "redirection"
+    name: "Redirect stdout"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for redirecting the output of a command to a file.
+      The `1>` operator is used to redirect the output of a command to a file.
+      But, as a special case, if the file descriptor is not specified before the operator `>`, the output is redirected to the standard output by default, so `>` is equivalent to `1>`.
+      You can read more about Redirecting output [here](https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Output).
```
Comment (andy1li) [inline course-definition.yml:770]: > Redirecting output

Perhaps all lowercase within the sentence could look better?

File: course-definition.yml [hunk 2]
```diff
@@ -751,3 +758,166 @@ stages:
...
+      In this stage, you'll implement support for redirecting the output of a command to a file.
+
+  - slug: "vz4"
+    primary_extension_slug: "redirection"
+    name: "Redirect stderr"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for redirecting the standard error of a command to a file.
+      The `2>` operator is used to redirect the standard error of a command to a file.
+      You can read more about Redirecting stderr [here](https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Output).
```
Comment (andy1li) [inline course-definition.yml:809]: Same as above.

File: course-definition.yml [hunk 3]
```diff
@@ -751,3 +758,166 @@ stages:
...
+
+  - slug: "el9"
+    primary_extension_slug: "redirection"
+    name: "Append stdout"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for appending the output of a command to a file.
+      The `1>>` operator is used to append the output of a command to a file.
+      As a special case, if the file descriptor is not specified before the operator `>>`, the output is redirected to the standard output by default, so `>>` is equivalent to `1>>`.
+      You can read more about Appending redirected output [here](https://www.gnu.org/software/bash/manual/bash.html#Appending-Redirected-Output).
```
Comment (andy1li) [inline course-definition.yml:847]: Same as above.

File: course-definition.yml [hunk 4]
```diff
@@ -751,3 +758,166 @@ stages:
...
+      In this stage, you'll implement support for appending the output of a command to a file.
+
+  - slug: "un3"
+    primary_extension_slug: "redirection"
+    name: "Append stderr"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for appending the standard error of a command to a file.
+      The `2>>` operator is used to append the standard error of a command to a file.
+      You can read more about Appending redirected output [here](https://www.gnu.org/software/bash/manual/bash.html#Appending-Redirected-Output).
```
Comment (andy1li) [inline course-definition.yml:892]: Same as above.

**Review by andy1li — APPROVED** [review]
LGTM! Just a nitpick on letter case.

File: course-definition.yml (hunk 1 above)
Comment (andy1li) [reply course-definition.yml:770]: Was the intention to make the title stand out a bit more? How about this:

```
Learn more about [Redirecting Output](https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Output).
```

Comment (ryan-gang) [reply course-definition.yml:770]: Yep, this sounds better!
Updated.

**Pull request merged**